app/controllers/user/two-factor-auth/post-configure.controller.js

```diff
@@ -5,7 +5,7 @@ const lodash = require('lodash')
 const paths = require('../../../paths')
 const userService = require('../../../services/user.service.js')
 const secondFactorMethod = require('../../../models/second-factor-method')
-const { RESTClientError } = require('../../../errors')
+const { RESTClientError } = require('@govuk-pay/pay-js-commons/lib/utils/axios-base-client/errors.js')
```

Comment:
Can we please get rid of the .js file extension - thanks

Reply:
Done.